crypto/evp_extra/evp_extra_test.cc

```diff
@@ -2627,19 +2627,57 @@ TEST_P(PerKEMTest, RawKeyOperations) {
   ASSERT_FALSE(EVP_PKEY_kem_check_key(pkey_new.get()));
 }
 
-TEST_P(PerKEMTest, KAT) {
+// Helper function that:
```

Comment:
why not modify a single line in the original function (https://github.com/aws/aws-lc/blob/main/crypto/evp_extra/evp_extra_test.cc#L2166) instead of copy-pasting the code? we shouldn't need the original `setup_ctx_and_generate_key` anymore, right?

Reply:
Offline discussion with Dusan, proposed another option in [333911c](https://github.com/aws/aws-lc/pull/1578/commits/333911c9922dbd8bcc6ae8e643c4b18638112adc) that combines both helpers into one.